commcare_connect/users/models.py

```diff
@@ -33,3 +44,28 @@ def get_absolute_url(self) -> str:
 
         """
         return reverse("users:detail", kwargs={"pk": self.id})
+
+
+class Organization(BaseModel):
+    name = models.CharField(max_length=255, unique=True)
+    slug = models.SlugField(max_length=255)
+
+    def save(self, *args, **kwargs):
+        self.slug = slugify(self.name)
```

Comment:
We should probably handle conflicts by appending a suffix to make sure it's unique - something like this: https://code.djangoproject.com/wiki/SlugifyUniquely